src/main.ts

```diff
@@ -326,6 +332,32 @@ class NixInstallerAction {
     }
   }
 
+  async flakehub_login(): Promise<string> {
+    const netrc_path = `${process.env["RUNNER_TEMP"]}/determinate-nix-installer-netrc`;
+
+    const jwt = await actions_core.getIDToken("api.flakehub.com");
+
+    await writeFile(
+      netrc_path,
+      [
+        `machine api.flakehub.com login flakehub password ${jwt}`,
+        `machine flakehub.com login flakehub password ${jwt}`,
+      ].join("\n"),
+    );
+
+    actions_core.info("Logging in to FlakeHub.");
+
+    // the join followed by a match on ^... looks silly, but extra_config
```

Comment:
Sometimes silly looking things make it into production. For example: Most of the code I write. :)

Reply:
🤭